app/src/components/common/form/dropdown/index.tsx

```diff
@@ -224,12 +226,19 @@ const Items = styled.div<{
   ${props => props.showScrollbar && DropdownScrollbarCSS}
 `
 
+const SecondaryText = styled.div`
+  color: #757575;
```

Comment:
This should be `color: ${props => props.theme.colors.textColor};`
Same for other colors.